lib/window.ts

```diff
@@ -41,7 +78,21 @@ export default function createWindow(currentWindow: Window, channel: Channel): W
 
   function updateHeight(height: number | null = null) {
     if (height === null) {
-      height = Math.ceil(document.documentElement.getBoundingClientRect().height)
+      const documentHeight = Math.ceil(document.documentElement.getBoundingClientRect().height)
+      const fullDocumentHeight = Math.max(
+        document.body.scrollHeight,
+        document.body.offsetHeight,
+        document.body.clientHeight,
+        document.documentElement.clientHeight,
+        document.documentElement.scrollHeight,
+        document.documentElement.offsetHeight
+      )
+
+      if (documentHeight !== fullDocumentHeight) {
+        height = absolutePositionedElems.size > 0 ? fullDocumentHeight : documentHeight
```

Comment:
What happens if we don't check for absolute elements here and always use `fullDocumentHeight`? Would this cause issues?

Reply:
This could work for some cases, but since the `offsetHeight` would consider all elements even if not visible, so what we do is that we actually check that absolute positioned elements are also visible before adding them to the Set.

But we could always return the `offsetHeight`, in that case we wouldn't have to check manually for absolute positioned elements, WDYT??
but it could cause some issues if there are elements with transform, as `offsetHeight` gets the dimensions before transform is applied.

Reply:
From what I see in the F36 source, we probably don't have any issue with `transform`. But I can see some third party libraries use `transform` to position dropdowns or context menus.

Could we iterate over all absolutely positioned elements, call `getBoundingClientRect` for each and then take the largest `bottom` value to calculate the height?